packages/vue-app/template/App.js

```diff
@@ -100,6 +101,28 @@ export default {
         }
       }
     },
+    async refreshPageData(showProgressBar) {
```

Comment:
As we call both asyncData and fetch, what do you think naming it one of these:
- `reload`,  `reloadPage` / `reloadRoute`
- `refrash`, `refrashPage` / `refrashRoute`
- `refetch`, `refetchPage` / `refetchRoute`

Reply:
Renamed to `refresh`:)